### ISSUE TITLE
[Bug]: Can't open portal file chooser: GDBus.Error:org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying

### ISSUE THREAD

**Issue by linlinxza** [flatpak/flatpak#5476]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a bug that matches the one I want to file, without success.
- [X] If this is an issue with a particular app, I have tried filing it in the appropriate issue tracker for the app (e.g. under https://github.com/flathub/) and determined that it is an issue with Flatpak itself.
- [X] This issue is not a report of a security vulnerability (see [here](https://github.com/flatpak/flatpak/blob/main/SECURITY.md) if you need to report a security issue).

### Flatpak version

1.12.7

### What Linux distribution are you using?

Other (specify below)

### Linux distribution version

KDE Neon

### What architecture are you using?

x86_64

### How to reproduce

1. Open Librewolf.
2. Right click and save as.
3. File picker not showing up.

### Expected Behavior

File picker showing up.

### Actual Behavior

File picker not show up. Instead, this shows up in the terminal when I try to select "Save Page as":

`(LibreWolf:2): Gtk-WARNING **: 08:13:24.731: Can't open portal file chooser: GDBus.Error:org.freedesktop.DBus.Error.NoReply: Message recipient disconnected from message bus without replying`

### Additional Information

I recently switched to Wayland. Strange thing is, all the other computers in the household use Wayland too. I just changed them to Wayland recently. The file picker works on those machines with Librewolf. Just not on my system.

EDIT:

This is specific to my user profile on Linux. I made a test profile and didn't have this issue. But... I really don't want to have to redo, or move things to another profile.

**Comment by linlinxza** [comment]
> > `Message recipient disconnected from message bus without replying`
> 
> This usually means "a D-Bus service crashed", in this case most likely `xdg-desktop-portal` or a backend implementation like `xdg-desktop-portal-kde`. Please check your system logs for crashes. If Neon has a crash dump collection framework like Abrt, Apport or systemd-coredump, getting the crash details from there would also be useful.
> 
> > This is specific to my user profile on Linux. I made a test profile and didn't have this issue
> 
> That's useful information, because it means it must be specific to something about your user profile that the test profile doesn't have.

I guess what I'll have to do is make a backup image of my OS drive or user folder. Then just put it in a VM and delete things one by one until I get it.

**Comment by linlinxza** [comment]
> Please check your system log before doing anything that time-consuming, it is likely to contain useful information.

It was very useful. I got it fixed. Apparently, it was saying that it was masked according to systemctl status xdg-desktop-portal.service. So I ran:

`systemctl --user unmask xdg-desktop-portal-gtk.service`

It didn't work.

So I did:

`systemctl --user mask xdg-desktop-portal-gtk.service`

And that fixed it. Man... So screwy. I never knew moving over to Wayland from X11 would be so hard. lol
